packages/docs/community/release-process/smart-contracts.md

```diff
@@ -21,11 +21,6 @@ Mixin contracts and libraries are considered part of the contracts that consume
 
 Each release is identified by a unique monotonically increasing number `N`, with `1` being the first release.
 
-### Contracts
```

Comment:
thought it would be smoother to segue into `git branches` / `git tags` first, since they are more relevant to `N` versioning mentioned just above.